spec/features/users/merchant_dashboard_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
```

Comment:
I figured since I was testing a merchant user's ability to see their shop's info it would go into users. But I ended up moving it's location to the merchants spec directory